src/components/utils/popover/popover-desktop.ts

```diff
@@ -226,18 +272,24 @@ export class PopoverDesktop extends PopoverAbstract {
   }
 
   /**
-   * Helps to calculate size of popover while it is not displayed on screen.
-   * Renders invisible clone of popover to get actual size.
+   * Helps to calculate parameters of popover that are only resolved when popover is displayed on screen.
+   * Renders invisible clone of popover to get actual values.
    */
   @cacheable
-  private get size(): {height: number; width: number} {
-    const size = {
-      height: 0,
-      width: 0,
+  public get renderParams(): {
```

Comment:
I think a separate `size()` getter was more clear. Why not just add a separate `containsInput` getter? Right now there are misunderstanding with `itemsRenderParams` of the popover constructor params

Reply:
if we make separate getter, we would have to render fake invisible popover twice

Reply:
Why not just call `this.nodes.popover(Dom.allInputsSelector)`?

Reply:
you're right, fixed